lookup_plugins/oo_option.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python2
+# -*- coding: utf-8 -*-
+# vim: expandtab:tabstop=4:shiftwidth=4
+
+'''
+oo_option lookup plugin for openshift-ansible
+
+Usage:
+
+    - debug:
+      msg: "{{ lookup('oo_option', '<key> default=<default_value>') }}"
```

Comment:
@lhuard1A Thank you for researching this and for looking up the behavior of other ansible lookup modules (in this case 'env').

I agree that this limitation is less then ideal. I'd prefer them to let us return None (seems reasonable, right?). But since they don't seem to right now, I prefer our code to behave consistently with the ansible code. In that way, our users will know what to expect.

I think the corner case where we actually want an empty string can be handled in other ways (like we'd handle them for any of the built in lookup plugins).

I'd prefer that we don't add the default option to the oo_option plugin, and instead use the default filter.

This makes our plugin behave consistently with the built in ansible lookup plugins.

@detiber what are your thoughts based on this new data?

@lhuard1A we can discuss this further in the ansible architecture meeting next week if you feel strongly about it.